dnf-behave-tests/dnf/downgrade.feature

```diff
@@ -24,10 +23,10 @@ Scenario: Downgrade one RPM
         | Action        | Package                                   |
         | downgrade     | flac-0:1.3.3-1.fc29.x86_64                |
    When I execute dnf with args "downgrade flac"
-   Then the exit code is 0
-    And stderr contains "Package flac of lowest version already installed, cannot downgrade it."
+   Then the exit code is 1
```

Comment:
dnf5 has the same behavior as dnf4 so during a downgrade if the package is already at the lowest version the exit code is 0.